src/include/OpenImageIO/fmath.h

```diff
@@ -1637,7 +1605,7 @@ inline OIIO_HOSTDEVICE float fast_correct_exp (float x)
     // much slower than the double version.  This was fixed in glibc 2.16.
     return static_cast<float>(std::exp(static_cast<double>(x)));
 #elif defined(__CUDACC__)
-    return static_cast<float>(exp(static_cast<double>(x)));
+    return static_cast<float>(std::exp(static_cast<double>(x)));
```

Comment:
You sure this one is right? Usually, plain old std::exp(x) would be the right thing. It's only for certain glibc libraries that for obscure reasons the double version is faster. 

I would think cuda would also fall into the "just use std::exp" case?

Reply:
Ah, I see. I will revert to the std::exp case.